release/release_tests.yaml

```diff
@@ -2733,6 +2745,8 @@
 
   cluster:
     byod:
+      pip:
```

Comment:
are these changes needed to run python 3.11 actually?

Reply:
this autorom is not compatible with 3.11, so I need to pull it out of the common Docker image

Reply:
should i add a TODO to track it? do we plan to add it back later?

Reply:
a TODO sounds good, thankks! we can add them back once autorom supports 3.11